components/observations/uploader/ObservationsUploader.ts

```diff
@@ -140,13 +244,18 @@ export class ObservationUploader {
       this.logger.debug({entry}, `processed task queue entry successfully`);
       this.subscribers.forEach(subscriber => subscriber(entry, true, entry.attemptCount));
       this.taskQueue.shift(); // we're done with this task, so remove it from the queue
+      entry.status = 'success';
+      this.completedTasks.push(entry);
```

Comment:
What consumes the contents of this? Just the routine dumping everything to logs? Do we expect this to be pruned ever?

Reply:
it's not saved to storage, so it's pruned from memory at app startup...I guess if someone is a very enthusiastic observation uploader then we might have a problem